package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "extract-text-webpack-plugin",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
Please revert this as we have release management in place and will take care of that

Reply:
understood